apm-sniffer/apm-sdk-plugin/sofarpc-plugin/src/main/java/org/apache/skywalking/apm/plugin/sofarpc/InvokeCallbackWrapper.java

```diff
@@ -20,13 +20,16 @@
 
 import com.alipay.remoting.InvokeCallback;
 import java.util.concurrent.Executor;
+import lombok.Getter;
 import org.apache.skywalking.apm.agent.core.context.ContextManager;
 import org.apache.skywalking.apm.agent.core.context.ContextSnapshot;
 import org.apache.skywalking.apm.agent.core.context.trace.AbstractSpan;
 
 public class InvokeCallbackWrapper implements InvokeCallback {
 
+    @Getter
```

Comment:
I think you should keep it in the `default` scope? This getter is generating public method.

Reply:
```suggestion
    @Getter(AccessLevel.PACKAGE)
```